### ISSUE TITLE
CentOS release 6.10 (Final)

### ISSUE THREAD

**Issue by gudge25** [sipcapture/heplify#186]
==1==
the binary is not working because there is no  GLIBC_2.14 in repo
# ./heplify
./heplify: /lib64/libc.so.6: version `GLIBC_2.14' not found (required by ./heplify)
./heplify: /lib64/libc.so.6: version `GLIBC_2.15' not found (required by ./heplify)
==2==
MAKE ERROR
# go version
go version go1.13.11 linux/amd64
# make
go build -ldflags "-s -w"  -o heplify *.go
go: github.com/bet365/jingo@v0.9.0: reading github.com/bet365/jingo/go.mod at revision v0.9.0: unknown revision v0.9.0
make: *** [all] Error 1
# 

Please help

**Comment by gudge25** [comment]
]# ldd --version
ldd (GNU libc) 2.12
Copyright (C) 2010 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
Written by Roland McGrath and Ulrich Drepper.
[root@asterisk heplify-1.56]#

**Comment by lmangani** [comment]
You should compile statically and/or use a static build. This is how we [build](https://github.com/sipcapture/heplify/blob/master/docker/heplify/Dockerfile) with musl inside alpine
```
CGO_ENABLED=1 GOOS=linux go build -a --ldflags '-linkmode external -extldflags "-static -s -w"' -o heplify .
```

**Comment by gudge25** [comment]
ok let me explaine U more 
==1== i tried  to wget  https://github.com/sipcapture/heplify/releases/download/1.56/heplify
then chmod  a+x heplify
then  ./heplify
BUT  GOT AN ERROR
[root@asterisk tmp]# ./heplify 
./heplify: /lib64/libc.so.6: version `GLIBC_2.14' not found (required by ./heplify)
./heplify: /lib64/libc.so.6: version `GLIBC_2.15' not found (required by ./heplify)
[root@asterisk tmp]#

==2==
i make git clone https://github.com/sipcapture/heplify.git
then cd /usr/src/heplify-1.56
then make
[root@asterisk heplify-1.56]# make
go build -ldflags "-s -w"  -o heplify *.go
go: github.com/bet365/jingo@v0.9.0: reading github.com/bet365/jingo/go.mod at revision v0.9.0: unknown revision v0.9.0
make: *** [all] Error 1

==3==
if i use   CGO_ENABLED=1 GOOS=linux go build -a --ldflags '-linkmode external -extldflags "-static -s -w"' -o heplify .
error seems the same
[root@asterisk heplify-1.56]# CGO_ENABLED=1 GOOS=linux go build -a --ldflags '-linkmode external -extldflags "-static -s -w"' -o heplify .
go: github.com/bet365/jingo@v0.9.0: reading github.com/bet365/jingo/go.mod at revision v0.9.0: unknown revision v0.9.0
You have new mail in /var/spool/mail/root

**Comment by lmangani** [comment]
I honestly have no idea, perhaps @negbie does

I can't reproduce this issue and I do see release seems to exist: https://github.com/bet365/jingo/releases

**Comment by negbie** [comment]
@gudge25 you have two problems. First is due to outdated os and glibc. imangani told you partly how to fix. So just build heplify the static way or update your os. Second problem is you are using an outdated version. Current version is 1.61 and there the jingo lib isn't used anymore. This indicates you aren't using go modules aswell. So update Go too.

**Comment by lmangani** [comment]
I managed to reproduce this finally. Reopening for a check.

**Comment by negbie** [comment]
@gudge25 @lmangani published version 1.62 under releases which is staticly build. Please test and reopen if necessary.
